app/build.gradle

```diff
@@ -30,5 +30,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    compile 'com.google.firebase:firebase-core:16.0.0'

```

Comment:
unused firebase dependency?
try to used "Implementation" for all dependencies.. since newest android studio has deprecated the uses of "compile" and "testImplementation" for declaring dependency.